bindings/java/src/org/sleuthkit/datamodel/TaggingManager.java

```diff
@@ -32,6 +32,10 @@
  * Provides an API to manage Tags.
  */
 public class TaggingManager {
+	// Score for tags with known bad status
+	private static Score BAD_SCORE = new Score(Score.Significance.NOTABLE, Score.MethodCategory.AUTO);
+	// Score for any other type of tag known status
+	private static Score UNKNOWN_SCORE = new Score(Score.Significance.LIKELY_NOTABLE, Score.MethodCategory.AUTO);
```

Comment:
This is already defined in Score or ScoringManager.    We should add the BAD_SCORE to there as well (and call it NOTABLE_SCORE)

Reply:
Hi @bcarrier ,
Are you saying you want me to use the constants I will be adding to `Score`?

Reply:
Yea, use the one already in Score and add the other there. Note that this PR will have conflicts with #2431, once that gets merged into develop.

Reply:
Ok.  Will do.